include/data_fifo.h

```diff
@@ -49,7 +49,7 @@ struct data_fifo {
  * @param data_fifo Pointer to the data_fifo structure
```

Comment:
This header is missing the opening definitions for doxygen groups. Please add them using the headers of previously added libraries as reference.

(Note: Closing tags are needed at the end of the file too.)